nbextensions/editorCell/main.js

```diff
@@ -147,16 +145,12 @@ define([
             var inputPrompt = this.element[0].querySelector('[data-element="prompt"]');
 
             if (inputPrompt) {
-                inputPrompt.innerHTML = div({
-                    style: { textAlign: 'center' }
-                }, [
-                    AppUtils.makeAppIcon(utils.getCellMeta(cell, 'kbase.editorCell.app.spec'))
-                ]);
+                inputPrompt.innerHTML = this.getIcon();
```

Comment:
Will this be impacted by not being centered in a div? My guess is no, and it seems to look ok.